addon/mixins/route.js

```diff
@@ -217,6 +217,10 @@ export default Ember.Mixin.create({
     return false;
   },
 
+  updateInfinityModel(model, infinityModel) {
+    model.pushObjects(infinityModel.get('content'));
+  },
+
```

Comment:
Psyched!  sorry for the nitpicks, just think this is an important addition to the API and I wanna get it right xo